x-pack/plugins/canvas/storybook/addon/src/components/action_list.tsx

```diff
@@ -51,16 +51,15 @@ export const ActionList: FC<{
   }, [recordedActions]);
 
   const options: EuiSelectableOption[] = Object.values(recordedActions).map((recordedAction) => ({
-    id: recordedAction.id,
     key: recordedAction.id,
     label: recordedAction.action.type,
     checked: recordedAction.id === selectedAction?.id ? 'on' : undefined,
   }));
 
   const onChange: (selectedOptions: EuiSelectableOption[]) => void = (selectedOptions) => {
     selectedOptions.forEach((option) => {
-      if (option && option.checked && option.id) {
-        const selected = recordedActions[option.id];
+      if (option && option.checked && option.key) {
+        const selected = recordedActions[option.key];
```

Comment:
EUI now disallows the `id` attr on these options because of how they are handled internally for a11y.
Let me know if this raises concerns and we can discuss